### PR TITLE
Add shape property configuration

### PR DESCRIPTION
**Pull request by faergeek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,6 +255,30 @@ import Demo, {props as P} from 'react-demo'
...
+
+<Demo
+  ...
+  props={{
+    post: {
+      title: P.string('My cool post'),
+
+      // and deeper
+      author: {
+        first_name: P.string('John'),
```
Comment (rpominov) [inline README.md]: Let's use camelCase here

File: examples/index.js [hunk 2]
```diff
@@ -43,6 +43,50 @@ const Description = React.createClass({
...
+          <small>By {author.first_name} {author.last_name}</small>
+        </p>
+
+        <p>{excerpt}</p>
+        {showLikes && <p onClick={onLikeClick}>Likes: {likeCount}</p>}
+      </div>
+    )
+  },
+
+})
```
Comment (rpominov) [inline examples/index.js:68]: I think there is no need for such complex example here, the most minimal one that only shows nesting will do (one or two props).

**Comment by rpominov** [comment]
All the rest looks good 👍

**Comment by faergeek** [comment]
Updated

**Pull request merged**